src/main/frontend/src/pages/Login.js

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import { login } from "../utils/userUtils";
+import classes from "./Login.module.css";
+import * as yup from 'yup';
+import { Formik, Form, Field } from 'formik';
+import { useAuth } from '../hooks';
+
+function Login() {
+  const [loginInfo, setLoginInfo] = useState({
+    email: "",
+    password: "",
+  });
+
+  const formValidation = yup.object().shape({
+    email: yup.string().required("*Email is required"),
+    password: yup.string().required("*Password is required"),
+  });
+
+  const[responseState, setResponseState] = useState("");
+  const{token, setToken} = useAuth('');
+
+  const { email, password } = loginInfo;
+
+  const onChange = (e) =>
+    setLoginInfo({ ...loginInfo, [e.target.name]: e.target.value });
+
+  const handleSubmit = async (user) => {
+    try {
+      console.log(user);
+      const person = await login(user);
+      if(person.status !== 200){
+        setResponseState("You entered wrong credidentials");
+      }else{
+      setToken(person.data)
+    }
+    } catch (e) {}
```

Comment:
On error, log error using console.error, and also setResponseState("Login request failed, please try again") to let user know something went wrong.